### PR TITLE
docs: add contribution guide and update readme

### PR DESCRIPTION
**Pull request by pythonpete32**
## Description

- add Contribution Guidelines
- update Readme

Task: [DX-59](https://aragonassociation.atlassian.net/browse/DX-59?atlOrigin=eyJpIjoiMWM4NDdjMWQ5ODcyNDJmMDkyY2Q3MzA0YTM5Mzk0YTYiLCJwIjoiaiJ9)

[DX-59]: https://aragonassociation.atlassian.net/browse/DX-59?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CONTRIBUTION_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- refactoring code,
+- or even writing new functionality to incorporate into the project.
+
+All members of our community are expected to follow our [Code of Conduct](https://discord.com/channels/672466989217873929/953988031713534022/1037740253080997918). Please make sure you are welcoming and friendly in all of our spaces. This is a non-negotiable 💙
+
+## Your first contribution
+
+Unsure where to begin contributing to Aragon?
+
+You can start with a [Good First Issue](https://github.com/aragon/developer-portal/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
```
Comment (Michael-A-Heuer) [inline CONTRIBUTION_GUIDE.md:21]: Good in theory, but in practice, we do it differently.
We are not using GH issues but JIRA for task planning. Accordingly external contributors won't see the tasks we are planning to do. The issue list is mostly empty for all our repos. There won't be labels with "Good first issue".

What's the plan for having good first issues here @Rekard0?

**Review by Michael-A-Heuer — COMMENTED** [review]
This contribution guide assumes that there are issues for certain tasks being labeled properly. However, our task planning and creation is generally done in JIRA.

Is the plan to move away from JIRA for the dev portal and use GH issues?

**Comment by pythonpete32** [comment]
> This contribution guide assumes that there are issues for certain tasks being labeled properly that contributors can take over. However, our task planning and creation is generally done in JIRA.
> 
> Is the plan to move away from JIRA for the dev portal and use GH issues?

no but the task requested to add contribution guidelines. I believe the goal is more to allow external people to fix small bugs or maybe to add tutorials (i know we dont have a tutorials section yet). JIRA will still be the tool we use internally. AFAIK our JIRA is private

**Comment by Michael-A-Heuer** [comment]
I don't see how this will lead to satisfying contributions if the tasks backlog is not visible to external collaborators and does not have "good first issue" labels.

Maybe an automated syncing between JIRA and developer portal related tasks is the answer. This is a general problem I see with all our repos: They are not really open/friendly to external collaborators because our backlog is not visible. 

The PR itself LGTM so I'll approve this PR.

**Review by Rekard0 — APPROVED** [review]

**Pull request closed without merging**